testing/participant/ParticipantShape.ttl

```diff
@@ -171,7 +171,7 @@ shapes:PersonShape
 	sh:property [
 		a sh:PropertyShape ;
 		sh:path ids:homepage ;
-		sh:datatype xsd:string ;
+    sh:nodeKind sh:IRI ;
```

Comment:
```suggestion
    sh:datatype xsd:anyURI ;
```

Should be anyURI instead.

Reply:
@changqin26

Reply:
Thanks for the hint, please review again.

Reply:
@JohannesLipp